From a4bcf807dda03657bd1757cc672fc40f3995f3d2 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sat, 12 Jun 2021 08:04:51 +0000
Subject: [PATCH] libstdc++: do not declare deque of const types

libstdc++ requires that value_type of std::deque is non-const.
---
 .../internal/execution/model_execution_manager_impl.cc          | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/segmentation_platform/internal/execution/model_execution_manager_impl.cc b/components/segmentation_platform/internal/execution/model_execution_manager_impl.cc
index 1e44eb1..7cab5fb 100644
--- a/components/segmentation_platform/internal/execution/model_execution_manager_impl.cc
+++ b/components/segmentation_platform/internal/execution/model_execution_manager_impl.cc
@@ -44,7 +44,7 @@ struct ModelExecutionManagerImpl::ExecutionState {
   SegmentationModelHandler* model_handler = nullptr;
   ModelExecutionCallback callback;
   base::TimeDelta bucket_duration;
-  std::deque<const proto::Feature> features;
+  std::deque<proto::Feature> features;
 };
 
 ModelExecutionManagerImpl::ModelExecutionManagerImpl(
-- 
2.31.1

